config/codestyle/eclipse-formatter.xml

```diff
@@ -59,7 +59,7 @@
         <setting id="org.eclipse.jdt.core.formatter.alignment_for_type_arguments" value="0"/>
         <setting id="org.eclipse.jdt.core.formatter.alignment_for_type_parameters" value="0"/>
         <setting id="org.eclipse.jdt.core.formatter.alignment_for_union_type_in_multicatch" value="16"/>
-        <setting id="org.eclipse.jdt.core.formatter.blank_lines_after_imports" value="1"/>
+        <setting id="org.eclipse.jdt.core.formatter.blank_lines_after_imports" value="3"/>
```

Comment:
this one we should discuss which value we want

Reply:
If no one has any issues with this, we can merge the changes in this PR and then revisit the rest as necessary.